cmd/lotus-storage-miner/info.go

```diff
@@ -355,3 +370,40 @@ func sectorsInfo(ctx context.Context, napi api.StorageMiner) error {
 
 	return nil
 }
+
+func producedBlocks(ctx context.Context, count int, head *types.TipSet, maddr address.Address, napi api.FullNode) error {
+	var err error
+	ts := head
+	fmt.Printf(" Epoch   | Block ID                                                       | Reward\n")
+	for count > 0 {
+		tsk := ts.Key()
+		bhs := ts.Blocks()
+		for _, bh := range bhs {
+			if bh.Miner == maddr {
+				rewardActor, err := napi.StateGetActor(ctx, reward.Address, tsk)
+				if err != nil {
+					return err
+				}
+
+				rewardActorState, err := reward.Load(cw_util.NewAPIIpldStore(ctx, napi), rewardActor)
+				if err != nil {
+					return err
+				}
+				blockReward, err := rewardActorState.ThisEpochReward()
```

Comment:
Isn't this per win?

Reply:
It is `State.ThisEpochReward` in reward. reference 
[https://github.com/filecoin-project/specs-actors/blob/0bcea9e3fa074b875135563cdaba6e786428c669/actors/builtin/reward/reward_actor.go#L101-L102](url)